packages/cli-template-monorepo-ethers/apps/web-app/.gitignore

```diff
@@ -1,9 +1,41 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+# dependencies
```

Comment:
If the new files/folders added are already in the main `.gitignore` file, then there is no need to add them here again. (for both templates: ethers and subgraph)

Reply:
Solved

- ethers: https://github.com/semaphore-protocol/semaphore/pull/841/commits/d20f5c82ece06a5ea303bbae93d7efccce9c238f
- subgraph: https://github.com/semaphore-protocol/semaphore/pull/841/commits/00e2403de15a85923929e24a5a14687ccf83f755